src/main/resources/data/minecraft/tags/items/chest_boats.json

```
@@ -1,6 +1,7 @@
 {
   "values": [
     "quark:blossom_chest_boat",
-    "quark:ancient_chest_boat"
+    "quark:ancient_chest_boat",
+    "quark:azalea_chest_boat",
```

With this trailing comma this is not valid JSON
```suggestion
    "quark:azalea_chest_boat"
```